userdocs/diagnostics/existential-any.md

```diff
@@ -1,24 +1,44 @@
-# `ExistentialAny`
+# Existential any (`ExistentialAny`)
 
-This diagnostic group includes errors and warnings pertaining to the `any` type
-syntax.
+`any` existential type syntax.
```

Comment:
I would so much prefer the book to just call them existential. The "boxed" in "boxed protocol type" is overloaded and uncatchy, and the "protocol" is just not accurate.